snapshots/devmapper/pool_device_test.go

```diff
@@ -247,7 +247,7 @@ func testDeactivateThinDevice(t *testing.T, pool *PoolDevice) {
 		err := pool.DeactivateDevice(context.Background(), deviceName, false, true)
 		assert.NilError(t, err, "failed to remove '%s'", deviceName)
 
-		assert.Assert(t, !pool.IsActivated(deviceName))
+		assert.Assert(t, pool.IsSuspended(deviceName))
```

Comment:
Why do we need `pool.IsSuspended` here? I wouldn't expect the device to be suspended after deactivation as we call `dmsetup remove` behind.
Can we leave `!pool.IsActivated(deviceName)` check in addition to make sure to returns false after deactivation.

Reply:
oh sorry, I forgot that pool.DeactivateDevice() do remove, pool.SuspendDevice() do suspend.
Done, thanks!